server/gfmd/db_ldap.c

```diff
@@ -3264,6 +3264,162 @@ gfarm_ldap_quota_dir_load(void *closure,
 	return (GFARM_ERR_FUNCTION_NOT_IMPLEMENTED);
 }
 
+/**********************************************************************/
```

Comment:
move the user_auth related functions to just aftert the user related functions,
to match the modified order of struct db_ops.